packages/core/src/render3/interfaces/view.ts

```diff
@@ -215,6 +216,11 @@ export interface LView extends Array<any> {
    * context.
    */
   [DECLARATION_VIEW]: LView|null;
+
+  /**
+   * More flags for this view. See LViewFlagsMore for more info.
+   */
+  [FLAGS_MORE]: LViewFlagsMore;
```

Comment:
I think we need a more descriptive name. What do you think of:

```suggestion
  [PREORDER_HOOK_FLAGS]: PreOrderHookFlags;
```